src/lib/util/Config.ts

```diff
@@ -88,7 +89,8 @@ const DEFAULT_CONFIG: ConfigContent = {
 				"**/*.cls",
 				"**/*.trigger"
 			],
-			disabled: false
+			disabled: false,
+			missingOptionsBehavior: MissingOptionsBehavior.WARN
```

Comment:
Setting the default behavior to "warn" seemed like it struck a proper balance of making sure the user was made aware of the new behavior without breaking everyone's existing integrations.